batch.py

```diff
@@ -16,7 +16,6 @@
 import click
 import logging
 
-from cloudpathlib import AnyPath
```

Comment:
are we 100% convinced AnyPath and to_path are interchangeable and we don't need this?